### DISCUSSION TITLE
Import/add images in costumes

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6081] [Ideas] [answered]
When you want to build costumes/backgrounds by using several images (something that I do a lot), as far as I know you must
1. import each image as a new costume/background
2. copy the image
3. paste the image in the costume/background where you are assembling all the images

By adding an "import" button in the costume/background editor (as it was in Scratch 1.4/2.0) the 3 steps would be shortened in just one step. 

By adding an "add" button in the costume/background editor (as it was in Scratch 2.0) the 3 steps would be shortened in just one step if you want to add a costume/background from the library

**Comment by WorldLanguages** [comment]
Is this #5083?

**Reply by sfederici** [reply, marked as answer]
I have no needs like [Chikorita-Lover](https://github.com/Chikorita-Lover)'s, but [mxmou's suggestion](https://github.com/ScratchAddons/ScratchAddons/issues/5083#issuecomment-1236862822) to solve [Chikorita-Lover](https://github.com/Chikorita-Lover)'s problem is exactly what I need. Removing everything from a costume and replacing it with a different image by importing it (the editor already has an easy to use delete button) it would quickly solve both mine and [Chikorita-Lover](https://github.com/Chikorita-Lover)'s problems, as far as I understand it.
